src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const properRobotKeys = Object.values(robot);
+  const properRobotValues = Object.keys(robot);
+  let properRobot = {};
+
+  for (let i = 0; i < properRobotKeys.length; i++) {
+    const robotValue = properRobotValues[i];
+    const robotKey = properRobotKeys[i];
+
+    if (properRobot.hasOwnProperty(robotKey)) {
+      return null;
+    }
+
+    properRobot = {
+      ...properRobot,
+      [robotKey]: robotValue,
+    };
+  }
```

Comment:
Honestly I would declare the properRobot with const and simply assign new property this way (naming is up to you obviously):
 ```suggestion
    properRobot[someKey] = someValue;
```